chrome/content/zotero/zoteroPane.xhtml

```diff
@@ -285,15 +303,9 @@
 									command="cmd_switchTextDirection"
 									key="key_switchTextDirection"
 									hidden="true" data-l10n-id="menu-edit-bidi-switch-text-direction"/>
-							<!--
-							#ifdef XP_UNIX
-							#ifndef XP_MACOSX
-							<menuseparator/>
-							<menuitem id="menu_preferences"
-								oncommand="Zotero.Utilities.Internal.openPreferences()" data-l10n-id="menu-preferences"/>
-							#endif
-							#endif
-							-->
+							<menuseparator id="menu_EditPreferencesSeparator"/>
+							<menuitem id="menu_EditPreferencesItem"
+								oncommand="Zotero.Utilities.Internal.openPreferences()" data-l10n-id="menu-settings"/>
```

Comment:
Ugh. Seems like Firefox, Chrome, and Edge all call it this now, but this is going to make cross-platform support annoying, both in documentation and the forums.

Reply:
Especially since we call the Connector Preferences that regardless of platform.

I was just fixing the localization key, since `menu-preferences` doesn’t seem to exist. No real reason we can’t stick with what we had. I’d imagine there’s other software that uses “Preferences” on Windows, and it’s what Zotero users are used to.

Reply:
Oh, wait, Ventura actually changes it to "Settings…" in the application menu too, along with the changes from "System Preferences" to "System Settings". So a little rough until everyone upgrades, but this is fine.